fhir-server/src/main/java/com/ibm/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -39,54 +41,54 @@
 /**
  * This class is a servlet filter which is registered with the REST API's servlet. The main purpose of the class is to
  * log entry/exit information and elapsed time for each REST API request processed by the server.
- * 
- * @author padams
  */
-public class FHIRRestServletFilter implements Filter {
+public class FHIRRestServletFilter extends HttpFilter {
+    private static final long serialVersionUID = 1L;
```

Comment:
are you actually serializing this class?  best to generate it not use 1l.

Reply:
I am not, it was just to make eclipse not complain about missing it (apparently HttpFilter is serializable whereas Filter was not)